gcc/rust/checks/errors/privacy/rust-privacy-reporter.cc

```diff
@@ -31,11 +32,61 @@ PrivacyReporter::PrivacyReporter (
     current_module (tl::nullopt)
 {}
 
+// Find a proc_macro, proc_macro_derive or proc_macro_attribute
+// attribute in a vector of attribute
+static tl::optional<std::string>
+find_proc_macro_attribute (const AST::AttrVec &outer_attrs)
+{
+  tl::optional<std::string> result;
```

Comment:
```suggestion
  tl::optional<std::string> result = tl::nullopt;
```

not sure which variant of the optional will get constructed since `std::string` has a default constructor, so let's make it explicit

Reply:
Fixed in [d5f8243](https://github.com/Rust-GCC/gccrs/pull/2446/commits/d5f82432f614bf2cd13b7b391fa0c2e379eedd69).